libs/form-builder/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@bedrockstreaming/form-builder",
-  "version": "0.8.4",
+  "version": "0.8.5-bump-nx.0",
   "dependencies": {
     "react-hook-form": "7.27.0"
   },
   "peerDependencies": {
-    "react": "17.0.2"
+    "react": "17 - 18"
```

Comment:
ℹ️ Interesting part here, this range should be satisfying since we don't seem to suffer from breaking changes